xclim/run_length.py

```diff
@@ -7,6 +7,16 @@
 from warnings import warn
 
 logging.captureWarnings(True)
+npts_opt = 9000
+
+
+def get_npts(da):
```

Comment:
Docstring

Reply:
I agree,  I do not completely understand why this doesn't scale efficiently.  I'm not sure we have any other explicit uses of ufuncs(?) ...